bedrock/firefox/templates/firefox/welcome/page4.html

```diff
@@ -0,0 +1,132 @@
+
+{# L10n: HTML page title #}
+{% block page_title %}{{ _('Download the Firefox Browser on your Mobile for iOS and Android') }}{% endblock %}
+
+{% block body_class %}{{ super() }} welcome-page4{% endblock %}
+
+{% set show_send_to_device = LANG in settings.SEND_TO_DEVICE_LOCALES %}
+
+{% set _entrypoint = 'mozilla.org-firefox-welcome-4' %}
+{% set _utm_campaign = 'welcome-4-mobile' %}
+
+{# L10n: the strong tag here is to underline Firefox. #}
+{% block content_intro %}
+{% call hero(
+    title=_('Wallet. Keys. Phone. <strong>Firefox.</strong>'),
+    desc=_('Take privacy with you on every device — and leave the data trackers behind.'),
+    class='mzp-t-firefox mzp-t-dark',
+    include_cta=True,
+    heading_level=1
+  ) %}
+
+  <p class="primary-cta">
+    <button class="js-modal-link mzp-c-button mzp-t-product" data-cta-position="primary" rel="external noopener">
+      {{ _('Get the Firefox App') }}
+    </button>
+  </p>
+
+  {% endcall %}
+
+  <div id="modal" class="mzp-u-modal-content mzp-l-content">
+    <h2 class="modal-logo">Firefox</h2>
+    <h3>{{ _('Get Firefox on your Phone') }}</h3>
+
+    {% if show_send_to_device %}
+      <p>{{ _('Send the download link right to your phone or email.') }}</p>
+      {{ send_to_device(include_title=False, class='vertical') }}
+    {% else %}
+      {# L10n: This string is reused from firefox/whatsnew #}
+      <p>{{ _('Download Firefox for your smartphone and tablet.') }}</p>
+      <div class="qr-code-wrapper">
+        <img src="{{ static('img/firefox/welcome/welcome-qr-firefox.png') }}" id="firefox-qr"
+          alt="QR code to scan for the Firefox mobile app.">
+      </div>
+    {% endif %}
+  </div>
+{% endblock %}
+
+{% block content_primary %}
+<div class="body-primary">
+  <img class="primary-image" src="{{ static('img/firefox/welcome/mobile-hero.png') }}" width="700" height="" alt="">
+  <p class="primary-image-desc">{{ _('“Firefox: Private, Safe Browser” on iOS or Android.') }}</p>
+</div>
+
+{% endblock %}
+
+{% block content_secondary %}
+<div class="c-picto-block t-adjacent-image">
+  <div class="c-picto-block-image">
+    <img src="{{ static('img/icons/gradient-shield.svg') }}" alt="">
+  </div>
+
+  {# L10n: "off your trail" is an expression for not being followed around. #}
+  <h3 class="c-picto-block-title">{{ _('Get data trackers off your trail') }}</h3>
+  <div class="c-picto-block-body">
+    <p>
+      {% set frende = ['en-US', 'en-GB', 'en-CA', 'de', 'fr', 'es-ES'] %}
+      {% if LANG in frende %}
+        {% trans privacy='https://www.mozilla.org/firefox/privacy/products/' %}
+        Enhanced Tracking Protection <a href="{{ privacy }}">blocks 2000+ trackers</a> from chasing you around the web.
+        {% endtrans %}
+      {% else %}
+        {% trans privacy='https://support.mozilla.org/en-US/kb/enhanced-tracking-protection-firefox-desktop' %}
+        Enhanced Tracking Protection <a href="{{ privacy }}">blocks 2000+ trackers</a> from chasing you around the web.
```

Comment:
Nit: as opposed to repeating the trans block here, could you just set the variable depending on locale?